hollow/src/main/java/com/netflix/hollow/api/producer/NoOpHollowProducerListener.java

```diff
@@ -0,0 +1,106 @@
+/*
+ *
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.hollow.api.producer;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * A trivial implementation of {@link HollowProducerListenerV2} which does nothing.
+ * Implementations of HollowProducerListenerV2 should subclass this class for convenience.
+ */
+public class NoOpHollowProducerListener implements HollowProducerListenerV2 {
```

Comment:
Small quibble. I wonder if `BaseHollowProducerListener` or `AbstractHollowProducerListener` would be better names? The only reason is that when I subclass it and do something in response to one or more signals, I'm not really a NoOp listener anymore.

Reply:
Good idea - I like BaseHollowProducerListener, since I wasn't planning on making it an abstract class.